.gitignore

```diff
@@ -333,4 +333,8 @@ ASALocalRun/
 node_modules
 
 # Package-lock
-package-lock.json
\ No newline at end of file
+package-lock.json

```

Comment:
We should check in the package-lock.json so we know the versions that work. We don't want to come back in a month's time and find this repo broken.